src/components/Property/Contacts/ContactsNoneAvailable.js

```diff
@@ -0,0 +1,15 @@
+const ContactsNoneAvailable = () => {
```

Comment:
Wonder if we can make this component more reusable such as `WarningText` and then pass the string value as props. Seems like nothing is custom apart from the text.

The reason I raise this is that one of the stories coming up will need to use the same warning text template. But happy to address this and refactor when the time comes

Reply:
Great shout. I've extracted a `WarningText` here: https://github.com/LBHackney-IT/repairs-hub-frontend/pull/89/commits/aa696b0ae5ed1cee3dcfe9dd037f37fbdc078c81